--- ../linux-source-3.2_0/scripts/package/builddeb	2014-12-16 18:17:40.778297011 +0200
+++ scripts/package/builddeb	2014-12-17 14:30:13.932840983 +0200
@@ -43,7 +43,7 @@
 	mips*)
 		debarch=mips$(grep -q CPU_LITTLE_ENDIAN=y $KCONFIG_CONFIG && echo el || true) ;;
 	arm*)
-		debarch=arm$(grep -q CONFIG_AEABI=y $KCONFIG_CONFIG && echo el || true) ;;
+		debarch=armhf ;; #$(grep -q CONFIG_AEABI=y $KCONFIG_CONFIG && echo el || true) ;;
 	*)
 		echo "" >&2
 		echo "** ** **  WARNING  ** ** **" >&2
